source/types/Case.ts

```diff
@@ -1,20 +1,21 @@
 import { EncryptionDetails } from "./Encryption";
 
-export type ApplicationStatusType =
-  | "active:ongoing"
-  | "active:processing"
-  | "active:signature:completed"
-  | "active:signature:pending"
-  | "active:submitted"
-  | "closed"
-  | "notStarted"
-  | "active:submitted:viva"
-  | "active:completionRequired:viva"
-  | "closed:approved:viva"
-  | "closed:completionRejected:viva"
-  | "closed:partiallyApproved:viva"
-  | "closed:rejected:viva"
-  | "notStarted:viva";
+export enum ApplicationStatusType {
+  ActiveOngoing = "active:ongoing",
+  ActiveProcessing = "active:processing",
+  ActiveSignatureCompleted = "active:signature:completed",
+  ActiveSignaturePending = "active:signature:pending",
+  ActiveSubmitted = "active:submitted",
+  Closed = "closed",
+  NotStarted = "notStarted",
+  ActiveSubmittedViva = "active:submitted:viva",
```

Comment:
This status is deprecated. Removed from api :)

Reply:
Which one? I can't see if you commented on just one line.

Reply:
Sorry, active:submitted:viva is not user in the api